### ISSUE TITLE
Compilation error with g++ 7.4.0 on Ubuntu 18.04.1

### ISSUE THREAD

**Issue by jjnsn** [Fushko/gammy#13]
Just a minor thing -- I installed Gammy on Ubuntu 18.04.1 and, in addition to the "PlaceholderText is not a member of QPalette" errors noted on the website, I got this one:

```
src/utils.h:102:1: error: body of constexpr function ‘constexpr T convertToRange(T, int, int, int, int) [with T = int]’ not a return-statement
 }
 ^
```

I fixed it by just doing this:

```
98 {
 99     return (((old_val - old_min) * (new_max - new_min)) / old_max - old_min) + new_min;
100 }
```

**Comment by Fushko** [comment]
Thanks for the heads-up. This is due to a restriction in C++11 which has been lifted in later versions.

**Comment by Fushko** [comment]
Should be fixed in 0.9.54.
